lego/apps/companies/serializers.py

```diff
@@ -36,9 +39,12 @@ def create(self, validated_data):
 
 
 class StudentCompanyContactSerializer(BasisModelSerializer):
+    user = PublicUserField(queryset=User.objects.all())
+    semester = SemesterField(queryset=Semester.objects.all())
```

Comment:
Are these querysets needed? If so, why

Reply:
no you're right they're not needed

Reply:
or after checking: for fields, you actually do need an explicity query set. Or else it causes an error

Reply:
hmm okay not sure if I get why - you'd think it would be redundant

Reply:
ik but that's how it's used throughout the whole code base